tests/unit/test_compiler.py

```diff
@@ -470,3 +470,144 @@ def test_set_operation(operation, sql, keywords):
 FROM t1"""
 
     assert result == query
+
+
+def test_geospatial_point():
```

Comment:
These are failing on ibis 2.x.

Would you mind creating a `tests/unit/test_compiler_geospatial.py` file with an importorskip and a minimum Ibis version? https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-importorskip

Reply:
It turns out that `ibis.geo_point()` is the historically-supported way of creating geo points, so I changed the unit test to use that.  I tested the change locally against Ibis 2.1.1 and it resolved the test failure.